lib/IRGen/IRGenModule.h

```diff
@@ -1430,6 +1430,14 @@ class IRGenModule {
   const char *getReflectionTypeRefSectionName();
   const char *getMultiPayloadEnumDescriptorSectionName();
 
+  /// Returns the special builtin types that should be emitted in the stdlib
```

Comment:
What makes these types "special" and sets them apart from other builtin types? Is this a techincal term used elsewhere, too?

Reply:
They're "special" because they're always emitted in the stdlib, whereas other builtins may or may not be emitted.